m2e-maven-runtime/org.eclipse.m2e.maven.indexer/pom.xml

```diff
@@ -19,13 +19,14 @@
 	</parent>
 
 	<artifactId>org.eclipse.m2e.maven.indexer</artifactId>
-	<version>1.18.0-SNAPSHOT</version><!--Keep it in sync with version in target-platform.target -->
-	<packaging>bundle</packaging>
+	<version>1.18.1-SNAPSHOT</version>
+	<packaging>eclipse-plugin</packaging>
 
 	<name>M2E Maven/Nexus Indexer</name>
 
 	<properties>
 		<guava.version>30.1-jre</guava.version>
+		<extraClasspathEntries><![CDATA[<classpathentry kind="src" path="src/main/java"/>]]></extraClasspathEntries>
```

Comment:
This one would ideally be inferred from the build.properties `source..` entry. Maybe add a comment here to kepe it in mind for future work.

Reply:
Yes, actually you are right. And you gave me the idea, that the `source..` element for the projects `build.properties` could be read using a `LoadProperties` Ant-Task and a subsequent additional for-loop could create src-classpathentry for each element. Then that special treatment is not required now and in case of future changes.
I'll try that.

Furthermore I think I should add another remark in the new README.md to state that the generated files should not be modified directly but only through the pom.xml.

Reply:
> 
> Yes, actually you are right. And you gave me the idea, that the `source..` element for the projects `build.properties` could be read using a `LoadProperties` Ant-Task and a subsequent additional for-loop could create src-classpathentry for each element. Then that special treatment is not required now and in case of future changes. I'll try that.

It worked.